app/services/articles/feeds/large_forem_experimental.rb

```diff
@@ -7,6 +7,15 @@ def initialize(user: nil, number_of_articles: Article::DEFAULT_FEED_PAGINATION_W
         @page = page
         @tag = tag
         @article_score_applicator = Articles::Feeds::ArticleScoreCalculatorForUser.new(user: user)
+
+        # Per #15292 we want to move towards not requiring feature's
+        # to have a main image.  This allows us to inch our way
+        # towards that configuration.
+        @must_have_main_image = false # FeatureFlag.accessible?(:featured_story_must_have_main_image)
+      end
+
+      def must_have_main_image?
+        @must_have_main_image
```

Comment:
Idle thought: I've considered adding an `attr_reader?` class method to generate these methods. Crystal has a `getter?` macro (it uses `getter`, `setter`, and `property` instead  of `attr_{reader,writer,accessor}`, respectively) to do this and it's really nice.

Reply:
That's a good idle thought!

Reply:
I like this too @jgaskins. Makes me want to add update my [really old gem](https://github.com/citizen428/ivar_encapsulation) to add a `predicate: true`) setting.